lib/cc/cli/path_fingerprint.rb

```diff
@@ -0,0 +1,33 @@
+require "digest/md5"
+
+module CC
+  module CLI
+    class PathFingerprint
+      def initialize(output)
+        @output = output
+      end
+
+      def compute
+        md5 = Digest::MD5.new
+        md5 << key
+        md5.hexdigest
+      end
+
+      def key
```

Comment:
It was nice to write assertions against the key in the spec. That's the only reason it's public.